composer.json

```diff
@@ -10,6 +10,50 @@
     {
       "type": "composer",
       "url": "https://packagist.org"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "plugins/wpml",
```

Comment:
Maybe we can use a more specific "vendor prefix", that indicates that these are "officially supported" plugins?